src/EditorFeatures/Core/Implementation/EditorLayerExtensionManager.cs

```diff
@@ -83,6 +86,32 @@ public override void HandleException(object provider, Exception exception)
 
                 _errorLoggerService?.LogException(provider, exception);
             }
+
+            private void LogEnableAndIgnoreProvider(object provider)
+            {
+                if (IsRoslynCodefix(provider))
+                {
+                    Log(Infobar_EnableCodefixAndIgnoreFutureErrors, provider.GetType().Name);
```

Comment:
hive I am not sure. but shell team might log that. probably need to dig to find that out. but roslyn version we do get. vs version also we do get. you can use http://vsdc to see all information being logged.